internal/multistorage/storage.go

```diff
@@ -109,7 +109,7 @@ func (ce *CloseError) Add(err error) {
 }
 
 func (ce *CloseError) Error() string {
-	errMsgs := make([]string, len(ce.specificStorageErrs))
```

Comment:
It is better, to set optional capacity to `len(ce.specificStorageErrs)` so it will look like `errMsgs := make([]string, 0, len(ce.specificStorageErrs))`

Reply:
fixed